packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -157,13 +157,13 @@ end
 function fbw.fetch_config(data)
     fbw.log('Fetch config from '.. json.stringify(data))
     local host = data.host
```

Comment:
either we avoid to define this local variable or later we use `host` instead of `data.host`